core/src/datatypes/account.ts

```diff
@@ -19,6 +19,22 @@ export type CreateAccountRequest = {
   contactMethods?: ContactMethod[];
 };
 
+export type AccountDataResponse = {
```

Comment:
this is missing a few pieces of account data. please add the user's status message, "do not disturb" setting, incoming friend requests, and outgoing friend requests.